Sources/Fuzzilli/Util/MockFuzzer.swift

```diff
@@ -54,6 +54,7 @@ class MockEnvironment: ComponentBase, Environment {
     var readPropertyNames = Set(["foo", "bar"])
     var writePropertyNames = Set(["foo", "bar"])
     var customPropertyNames = Set(["foo", "bar"])
+    var customMethodNames = Set(["foo", "bar"])
```

Comment:
Nit: Doesn't this violate the invariant that these two sets should be disjoint? Maybe just call it `m1`, `m2` or so?